packages/langium-cli/jest.config.js

```diff
@@ -7,7 +7,8 @@
 module.exports = {
     preset: 'ts-jest',
     roots: [
-        '<rootDir>/src'
+        '<rootDir>/src',
+        '<rootDir>/test',
```

Comment:
Suggestion: We don't actually use this. Can you delete this file, while you're already at it?

Reply:
Yes